From 4c8d63d47a3c863e31fb4436fce57da0510d6cd6 Mon Sep 17 00:00:00 2001
From: Alfredo Moralejo <amoralej@redhat.com>
Date: Tue, 9 Jul 2024 08:56:39 +0200
Subject: [PATCH] Disable warnings on sqlalchemy 2.0 in tests

---
 tox.ini | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tox.ini b/tox.ini
index 399dd2b..1523ade 100644
--- a/tox.ini
+++ b/tox.ini
@@ -16,8 +16,8 @@ setenv =
   {postgresql,all}: PIFPAF_POSTGRESQL=pifpaf -g OS_TEST_DBAPI_ADMIN_CONNECTION run postgresql --
   {mysql,all}: PIFPAF_MYSQL=pifpaf -g OS_TEST_DBAPI_ADMIN_CONNECTION run mysql --
   {mysql,postgresql,all}: BASECOMMAND={toxinidir}/tools/run-pifpaf-tests.sh
-# TODO(stephenfin): Remove once we bump our upper-constraint to SQLAlchemy 2.0
-  SQLALCHEMY_WARN_20=1
+  SQLALCHEMY_WARN_20=0
+  SQLALCHEMY_SILENCE_UBER_WARNING=1
 deps =
   -c{env:TOX_CONSTRAINTS_FILE:https://releases.openstack.org/constraints/upper/master}
   -r{toxinidir}/test-requirements.txt
-- 
2.45.1

